### ISSUE TITLE
Getting ModuleNotFoundError: No module named 'requests' on Elastic Beanstalk

### ISSUE THREAD

**Issue by hcNick** [jpetrucciani/hubspot3#35]
```
Using cached https://files.pythonhosted.org/packages/78/d7/3e3a9ff716b29c07b451bcd200449cdf360b15415f9bd17c162e1b33460c/hubspot3-3.2.11.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-mn7fko9u/hubspot3/setup.py", line 5, in <module>
        from hubspot3.globals import __version__
      File "/tmp/pip-build-mn7fko9u/hubspot3/hubspot3/__init__.py", line 5, in <module>
        from hubspot3.error import HubspotBadConfig, HubspotNoConfig
      File "/tmp/pip-build-mn7fko9u/hubspot3/hubspot3/error.py", line 6, in <module>
        from hubspot3.utils import force_utf8
      File "/tmp/pip-build-mn7fko9u/hubspot3/hubspot3/utils.py", line 4, in <module>
        import requests
    ModuleNotFoundError: No module named 'requests'
```

Getting this on Elastic Beanstalk, if I ```pip install requests``` separately and then run it all works fine?

**Comment by jpetrucciani** [comment]
Ah yeah I see the issue here - I'll take a look into this soon.

Thanks for the report!

**Comment by hcNick** [comment]
No worries, thanks!

**Comment by jpetrucciani** [comment]
This should be fixed now on version [`3.2.13`](https://pypi.org/project/hubspot3/3.2.13/) on pip - thanks again for bringing this issue up!
